apps/src/javalab/JavalabButton.jsx

```diff
@@ -64,19 +65,9 @@ JavalabButton.defaultProps = {
 };
 
 const styles = {
-  button: {
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
-    color: color.white
-  },
-  horizontal: {
-    flexDirection: 'row'
-  },
   disabled: {
     backgroundColor: color.light_gray,
     borderColor: color.light_gray,
-    ':hover': null,
```

Comment:
Was this not doing anything?

Reply:
Yep, traced it back to the PR where it was added. The consumer it was added for no longer uses JavalabButton, and other uses of disabled JavalabButtons didn't rely on it.